app/src/main/java/org/systers/mentorship/view/fragments/ProfileFragment.kt

```diff
@@ -20,7 +36,25 @@ class ProfileFragment : BaseFragment() {
          * Creates an instance of ProfileFragment
          */
         fun newInstance() = ProfileFragment()
+
+        /**
+         * Binding adapter used for setting the right background for EditTexts in this fragment's layout.
+         */
+        @JvmStatic
+        @BindingAdapter("android:background")
```

Comment:
Can't see where you are using it in layouts

Reply:
It's automatically used in `TextInputEditText`s, when I set `android:background="${editing}"`

Reply:
Found it 👍